tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/endpoints/CamelSend.java

```diff
@@ -0,0 +1,56 @@
+package io.hawt.tests.features.pageobjects.fragments.camel.tabs.endpoints;
+
+import static com.codeborne.selenide.Condition.enabled;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.byAttribute;
+import static com.codeborne.selenide.Selectors.byTagAndText;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+import io.hawt.tests.features.pageobjects.pages.camel.CamelPage;
+
+
+/**
+ * Represents Send Tab page in Camel Endpoints.
+ */
+public class CamelSend extends CamelPage {
+
+    /**
+     * Add one header to the message.
+     *
+     * @param header      name to be added
+     * @param headerValue to be added
+     */
+    public void addOneHeader(String header, String headerValue) {
+        $(byTagAndText("button", "Add Headers")).shouldBe(enabled).click();
+        $(byAttribute("aria-label", "Search input")).shouldBe(enabled).click();
+        $(byTagAndText("p", header)).shouldBe(visible).click();
+        $(byXpath("//input[@aria-label='value-input-0']")).shouldBe(enabled).setValue(headerValue);
```

Comment:
If this is not yet just defined on the element, consider defining some more meaningful value to it for hawtio-next.
> aria-label='value-input-0'

Reply:
Ok, thanks for pointing to this. I will fix it

Reply:
Once you send a pull req to hawtio-next, I can quickly release a new version of it so you can use it here.

Reply:
I have fixed and it seems all checks have passed. 
If it's Ok now, can you merge it, please?
Thanks